### PR TITLE
Add tag support to list view

### PR DESCRIPTION
**Pull request by thornbill**
**Changes**
* Adds tag support to the list view
* Replaces the search link for tags with the list view link

**Issues**
N/A

### CODE REVIEW

**Comment by viown** [comment]
https://github.com/jellyfin/jellyfin-web/blob/d01cfa498b5f3d8b8fb3996aa1cf4949b7cd0bd6/src/controllers/list.js#L1211

Maybe we can add `params.tag` here so the title and year are displayed.

**Comment by thornbill** [comment]
Thanks for the feedback @viown! I made some adjustments to add `tag` as the `type` as I noticed the default sort order was also weird without specifying a `type`.

File: src/controllers/itemDetails/index.js [hunk 1]
```diff
@@ -1309,8 +1309,13 @@ function renderTags(page, item) {
     }
 
     tags.forEach(tag => {
+        const href = appRouter.getRouteUrl('tag', {
+            context,
```
Comment (grafixeyehero) [inline src/controllers/itemDetails/index.js:1313]: removed unused context

**Review by grafixeyehero — COMMENTED** [review]
LGTM 🚀

**Review by grafixeyehero — APPROVED** [review]

**Pull request merged**